static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -301,7 +301,8 @@ const CodeWrapper = styled('div')`
   position: relative;
   padding: 0;
 
-  && pre {
+  && pre,
+  && code {
```

Comment:
Can this also be applied globally? Then it can apply to breadcrumb pre/code as well which is also using size 13.

Reply:
Talking to @vuluongj20 we do want 13px by default (to match the size of `fontSizeMedium`), but for the stack trace everything is `fontSizeSmall` so we want the code font size to be smaller to match. So we'll need to adjust manually for these cases.